pages/common/cradle.md

```diff
@@ -0,0 +1,20 @@
+# cradle
+
+> The Cradle PHP framework.
+> See `cradle-install`, `cradle-deploy` and other pages for additional information.
+
+- Connect to a server:
+
+`cradle connect {{connection_name}}`
```

Comment:
I am guessing this is something like "user:password@hostname:port" ? Can we write something like that to make it clearer ?

Reply:
It's just something like `app` or whatever you want it to be called. The details for it are specified in the `config/deploy.php` file.

https://github.com/CradlePHP/Cradle/blob/master/config/deploy.php